src/app/find/errorReports/ErrorReportText.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Header from '../../../components/header';
+import Button from '../../../components/button';
+import TextArea from '../../../components/textarea';
+
+const ErrorReportText = (props) => {
+  const {
+    errorReportText,
+    onChange,
+    onSubmit,
+    goToViewLocation,
+  } = props;
+
+  return (
+    <div>
+      <Header size="large" className="locationTitle">Error Report</Header>
+      <div className="mx-4 mb-3 text-left">
+        <div className="w-100 mr-5">
+          <Header size="medium" className="mb-3">
+            Please provide the correction below
+            <br />
+            (Please don&#39;t enter any private information)
+          </Header>
+        </div>
+      </div>
+      <div>
+        <div className="mx-4 mb-1">
+          <TextArea
+            value={errorReportText}
+            onChange={onChange}
+            minRows={7}
+            autoFocus
+            fluid
+            className="border"
+          />
+        </div>
+        <div className="mx-4 mb-1">
+          <Link to={`${props.match.url}/thanks`}>
```

Comment:
Same comment as in the other component: This is really the 1 thing that _should_ be handled by the container, rather than having a Button that doesn't really do anything wrapped in a link.

It just never would have occurred to me to look here for how navigation happens, when in every other respect this is entirely a presentational component.